test/cbmc/proofs/Jobs_Describe/Makefile

```diff
@@ -7,8 +7,8 @@ HARNESS_FILE=Jobs_Describe_harness
 # to provide 100% coverage and to reduce unwindings
 THINGNAME_MAX_LENGTH=6
 EXCEED_THINGNAME_MAX_LENGTH=7
-JOBID_MAX_LENGTH=70
-EXCEED_JOBID_MAX_LENGTH=71
+JOBID_MAX_LENGTH=64
+EXCEED_JOBID_MAX_LENGTH=65
```

Comment:
Oops, I think I committed that 70 and 71 by accident.  The earlier version had much smaller values, like 4 and 5.  Whichever values we use, the UNWINDSET below needs to change if EXCEED_JOBID_MAX_LENGTH is greater than EXCEED_THINGNAME_MAX_LENGTH.

Reply:
Changed to 6 and 7 like thingname.